GetRawYieldsDplusDs.C

```diff
@@ -175,19 +179,30 @@ int GetRawYieldsDplusDs(int cent, bool isMC, TString infilename, TString cfgfile
 
     TString massaxistit = "";
     if(particle==kDplus) massaxistit = "#it{M}(K#pi#pi) (GeV/#it{c}^{2})";
+    else if(particle==kD0) massaxistit = "#it{M}(K#pi) (GeV/#it{c}^{2})";
     else if(particle==kDs) massaxistit = "#it{M}(KK#pi) (GeV/#it{c}^{2})";
     else if(particle==kLc) massaxistit = "#it{M}(pK^{0}_{s}) (GeV/#it{c}^{2})";
     else if (particle==kDstar) massaxistit = "#it{M}(pi^{+}) (GeV/#it{c}^{2})";
     
     //load inv-mass histos
     auto infile = TFile::Open(infilename.Data());
     if(!infile || !infile->IsOpen()) return -1;
+    auto infileref = TFile::Open(refFileName.Data());
+    if(!infileref || !infileref->IsOpen()) return -1;
```

Comment:
This should be conditional on `enableRef`, otherwise the script would not work for the other particles